src/core/server/elasticsearch/cluster_client.test.ts

```diff
@@ -165,6 +165,17 @@ describe('#callAsInternalUser', () => {
     ).rejects.toStrictEqual(mockAuthenticationError);
   });
 
+  test('sets the onabort property of a signal if provided', () => {
+    const controller = new AbortController();
+
+    clusterClient.callAsInternalUser('ping', undefined, {
+      wrap401Errors: false,
+      signal: controller.signal,
+    });
+
+    expect(typeof controller.signal.onabort).toBe('function');
```

Comment:
Instead of testing the AbortSignal itself, can we call `abort` and ensure the call fails?